docs/specs/openapi.yaml

```diff
@@ -0,0 +1,948 @@
+      - FULL TEXT SEARCH
+      summary: Provides all entities related to indom
+      description: | 
+        It provides all entities (instances, metrics) related to indom, including itself, that is passed to the server via the *query* parameter.
+
+        **Reference:** [pmSearchTextInDom](https://man7.org/linux/man-pages/man3/pmSearchTextInDom.3.html)(3)
+      operationID: get/search/indom
+      parameters:
+      - name: query
+        in: query
+        description: Search indom for search engine
+        schema:
+          type: string
+      - name: offset
+        in: query
+        description: Result offset cursor for pagination
+        schema:
+          type: number
+      - name: limit
+        in: query
+        description: M results to include in response
+        schema:
+          type: number
+
+  /search/info:
+    get:
+      tags:
+      - FULL TEXT SEARCH
+      summary: Provides metrics relating to operation
+      description: | 
+        It provides metrics relating to operation of the search engine, in particular showing document and text record counts.
+        ```
+        $ curl -s http://localhost:44322/search/info | pmjson
+        ```
+        **Reference:** [pmSearchInfo](https://man7.org/linux/man-pages/man3/pmSearchInfo.3.html)(3)
+      operationID: get/search/info
+      responses:
+        200:
+          description: successful operation
+          content:
+            pmjson:
+              schema:
+                type: object
+                properties:
+                  docs:
+                    type: number
+                  terms:
+                    type: number
+                  records:
+                    type: number
+                  records_per_doc_avg:
+                    type: number
+                  bytes_per_record_avg:
+                    type: number
+                  inverted_sz_mb:
+                    type: number
+                  inverted_cap_mb:
+                    type: number
+                  inverted_cap_ovh:
+                    type: number
+                  skip_index_size_mb:
+                    type: number
+                  score_index_size_mb:
+                    type: number
+                  offsets_per_term_avg:
+                    type: number
+                  offset_bits_per_record_avg:
+                    type: number
+                example:
+                  docs: 1589
+                  terms: 3855
+                  records: 116831
+                  records_per_doc_avg: 73.52
+                  bytes_per_record_avg: 6.36
+                  inverted_sz_mb: 0.71
+                  inverted_cap_mb: 0.00
+                  inverted_cap_ovh: 0.00
+                  skip_index_size_mb: 0.00
+                  score_index_size_mb: 0.00
+                  offsets_per_term_avg: 9.41
+                  offset_bits_per_record_avg: 8.00
+    
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+components:
+  schemas:
+    response1:
```

Comment:
Can you give them names? For example "seriesIds"

Reply:
Done!